File: Backend/Spring/TicketManagementSystem_Spring_Backend/src/main/java/org/practica/exceptions/NoAuthorizationException.java
package org.practica.exceptions;

public class NoAuthorizationException extends Exception {
    private String message;

    public NoAuthorizationException(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: Backend/.NET/TicketManagementSystem_NET_Backend/TicketManagementSystem_NET_Backend/Security/NoAuthorizationException.cs
using System.Runtime.Serialization;

namespace TicketManagementSystem_NET_Backend.Security
{
    [Serializable]
    internal class NoAuthorizationException : Exception
    {
        public NoAuthorizationException()
        {
        }

        public NoAuthorizationException(string? message) : base(message)
        {
        }

        public NoAuthorizationException(string? message, Exception? innerException) : base(message, innerException)
        {
        }

        protected NoAuthorizationException(SerializationInfo info, StreamingContext context) : base(info, context)
        {
        }
    }
}